x-pack/plugins/security_solution/public/common/components/event_details/response_actions_view.tsx

```diff
@@ -36,61 +65,69 @@ export const useResponseActionsView = <T extends object = JSX.Element>({
   ecsData?: Ecs | null;
   rawEventData: SearchHit | undefined;
 }): EuiTabbedContentTab | undefined => {
+  const viewData = useMemo(
+    () => ({
+      id: EventsViewType.responseActionsView,
+      'data-test-subj': 'responseActionsViewTab',
+      name: i18n.RESPONSE_ACTIONS_VIEW,
+    }),
+    []
+  );
   const responseActionsEnabled = useIsExperimentalFeatureEnabled('endpointResponseActionsEnabled');
   const expandedEventFieldsObject = rawEventData
     ? (expandDottedObject((rawEventData as RawEventData).fields) as ExpandedEventFieldsObject)
     : undefined;
 
   const responseActions =
     expandedEventFieldsObject?.kibana?.alert?.rule?.parameters?.[0].response_actions;
-  const shouldEarlyReturn = !rawEventData || !responseActionsEnabled || !responseActions?.length;
+  const shouldEarlyReturn = !rawEventData || !responseActionsEnabled;
 
   const alertId = rawEventData?._id ?? '';
+  const [isLive, setIsLive] = useState(false);
 
   const { data: automatedList, isFetched } = useGetAutomatedActionList(
     {
       alertIds: [alertId],
     },
-    { enabled: !shouldEarlyReturn }
+    { enabled: !shouldEarlyReturn, isLive }
   );
 
-  if (shouldEarlyReturn) {
-    return;
-  }
-
-  const ruleName = expandedEventFieldsObject?.kibana?.alert?.rule?.name?.[0];
-
-  const totalItemCount = automatedList?.items?.length ?? 0;
+  useLayoutEffect(() => {
+    setIsLive(() => !(!responseActions?.length || !!automatedList?.items?.length));
+  }, [automatedList, responseActions?.length]);
```

Comment:
Could you elaborate on what this does?

Reply:
And why it's in a layout effect

Reply:
Sure, while working on tests I found out that the alert sometimes do not contain our actions. Probably it's just an edge case that we were able to reproduce in cypress but in general the behavior:
- when we have response actions and we do not have items fetched in the automatedList, we want to set isLive to true. If we meet the condition where we have actions and we have items fetched we set to false. 
Whenever it's `true` , we set interval for refetch to 5000ms thus React-query will refetch the data. 
This way we provide a better UX where user do not have to rerender the component in order to fetch data again.

Reply:
Regarding `useLayoutEffect` I think it's not required here. I'll change to useEffect 👍